cclunit-framework-source/doc/CCLUTMOCKING.md

```diff
@@ -0,0 +1,295 @@
+
+# CCL Unit Mocking
```

Comment:
I just looked at the pretty view of [this file](https://github.com/cerner/cclunit-framework/blob/cef8d0867cbeebf3bb9adb3498bc31500d0fc48c/cclunit-framework-source/doc/CCLUTMOCKING.md]) for the first time.  It seems like it would be better to have a brief high-level description of  what CCL Unit Mocking is all about in the location of the warning and put the warning after that. It might also be good to have a TOC just below the description making it easy to jump directly to the warning, spec, implementation notes and example.